dom.bs

```diff
@@ -2195,6 +2195,10 @@ steps:</p>
  <li><p>If the <i>open flag</i> is set and <var>shadow</var>'s <a for=ShadowRoot>mode</a> is
  <em>not</em> "<code>open</code>", then return null.</p></li>
 
+ <li><p>If the <var>shadow</var>'s <a for=ShadowRoot>slot assignment</a> is set to "<code>manual</code>",
+ return the associated slot in <var>shadow</var>'s tree whose <a>manuallyAssignedNodes</a> includes
```

Comment:
You want to change this to be more similar to the next step in terms of wording. Although I suppose you can leave out "first" and "tree order" since there's guaranteed to be a single slot if I understand it correctly.

Reply:
Updated wording.